homeassistant/components/niko_home_control/__init__.py

```diff
@@ -1 +1,37 @@
-"""The niko_home_control component."""
+"""The Niko home control integration."""
+
+from __future__ import annotations
+
+from nikohomecontrol import NikoHomeControlConnection
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .errors import CannotConnect
+
+PLATFORMS: list[str] = ["light"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
Extend ConfigEntry type to add the type of `entry.runtime_data` and use that instead of `hass.data`

Reply:
Sounds like Chinese 😂